### PR TITLE
Wbs4.2.7/user profile team

### PR DESCRIPTION
**Pull request by getnithesh**
- List teams
- Create a new team
- Assign and remove members to /from a team
- Delete team

### CODE REVIEW

**Review by chrisweilacker — COMMENTED** [review]
This branch has conflicts that must be resolved please merge in development locally and get any conflicts resolved.  Thanks.

File: src/components/UserManagement/ResetPasswordUserPopup.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react'
```
Comment (Junhao6503) [inline src/components/UserManagement/ResetPasswordUserPopup.jsx:1]: What's the difference between this file and the old ResetPasswordPopup?

File: src/components/UserManagement/StatusUserPopup.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```
Comment (Junhao6503) [inline src/components/UserManagement/StatusUserPopup.jsx:1]: Has this component been used? I did not find a button to trigger this popup.

File: src/components/UserManagement/ResetPasswordUserPopup.jsx (hunk 1 above)
Comment (getnithesh) [reply src/components/UserManagement/ResetPasswordUserPopup.jsx]: Removed this file, this is the wrong one which is accidentally included.

File: src/components/UserManagement/StatusUserPopup.jsx (hunk 2 above)
Comment (getnithesh) [reply src/components/UserManagement/StatusUserPopup.jsx]: Same here, wrong one, accidentally included.

File: src/components/Teams/TeamMembersPopup.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                  <th>#</th>
+                  <th>User Name</th>
+                  <th>
+                    {' '}
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                {props.members.teamMembers.length > 0
+                  ? props.members.teamMembers.map((user, index) => (
```
Comment (Junhao6503) [inline src/components/Teams/TeamMembersPopup.jsx:67]: Each child in a list should have a unique "key" prop.

**Review by Junhao6503 — CHANGES_REQUESTED** [review]

File: src/components/Teams/TeamMembersPopup.jsx (hunk 3 above)
Comment (getnithesh) [reply src/components/Teams/TeamMembersPopup.jsx:67]: Corrected

**Review by Junhao6503 — DISMISSED** [review]
looks good now

**Review by Junhao6503 — APPROVED** [review]
looks good for me

**Pull request merged**